spec/features/edit_beer_spec.rb

```diff
@@ -0,0 +1,71 @@
+require "rails_helper"
+
+feature 'edit beer information', %{
+  As the original beer poster,
+  I want to be able to edit the information for the beer that I’ve added
+  so that people think I’m competent
+} do
+  # Acceptance Criteria
+  # * I must be signed in
+  # * I must be able to get here through the details page for a specific beer
+  # * I must fill in the required fields
+  # * I must be presented with errors if I fill out the form incorrectly
+
+  scenario "sucessfully edited" do
+    beer = FactoryGirl.create(:beer)
+    user = beer.user
+
+    visit root_path
+    sign_in_as(user)
+    click_link beer.name
+    click_link "Edit Beer"
+
+    fill_in "Name", with: "Edited Beer"
+    fill_in "ABV", with: "50"
+    fill_in "IBU", with: "100"
+    select "Lager", from: "Style"
+    click_button 'Update Beer'
+
+    expect(page).to have_content('Beer Information Updated')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.